src/main/java/by/dragonsurvivalteam/dragonsurvival/common/entity/DragonEntity.java

```diff
@@ -45,21 +45,14 @@ public class DragonEntity extends LivingEntity implements IAnimatable, CommonTra
 	public final ArrayList<Double> headPitchAverage = new ArrayList<>();
 	public final ArrayList<Double> tailSideAverage = new ArrayList<>();
 	public final ArrayList<Double> tailUpAverage = new ArrayList<>();
-	/**
-	 * This reference must be updated whenever player is remade, for example, when changing dimensions
-	 */
-	public volatile Integer player;
+	/** This reference must be updated whenever player is remade, for example, when changing dimensions */
+	public volatile Integer playerId; // TODO :: Use string uuid?
```

Comment:
Not quite sure what you want to do here, but for reference: skins MUST work by nickname, not uuid.  We used to have drafts for skins by uuid, but they don't work with the offline mod and in general I and the players want skins to be tied to nickname, not uuid